src/Symfony/Component/Validator/Tests/Constraints/LengthValidatorTest.php

```diff
@@ -112,6 +112,13 @@ public static function getThreeCharactersWithWhitespaces()
         ];
     }
 
+    public static function getOneGraphemeWithThirteenBytesAndFourCodePoints()
```

Comment:
Using a data provider for a single provided case looks quite weird to me. I would inline the value in each tests (or use a private constant if duplicating the emoji needs to be avoided, but I don't think we need this).

Currently, the code is quite bad because tests using that data provider are actually making very strong assumption about the value being provided, making it unlikely to ever be able to provide more cases in it.